recipes/clyngor/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I don't see a license in the upstream repo, so I'm assuming you just grabbed a generic copy from somewhere online? This is slightly problematic since the author does not specify which version of GPL they want. The repo does include a `MANIFEST.in` and that includes `LICENSE`, but it's not there in the repository. I've raised an issue here:

https://github.com/Aluriak/clyngor/issues/26